tests/session/session-startTransaction-001.phpt

```diff
@@ -0,0 +1,23 @@
+--TEST--
+MongoDB\Driver\Session::startTransaction() mini test
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php skip_if_server_version('<', '4.0'); ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$manager = new MongoDB\Driver\Manager(URI);
+$session = $manager->startSession();
+
+$session->startTransaction();
+$session->abortTransaction();
+
+$session->startTransaction();
+$session->commitTransaction();
```

Comment:
I tested these methods against a 4.0.0-rc5 standalone and there were no errors in any of the transaction tests.

With regard to this test alone (I haven't reviewed the others yet), that may be due to the fact that a transaction is not actually started until the first command executed within in. See [`startTransaction`](https://github.com/mongodb/specifications/blob/master/source/transactions/transactions.rst#starttransaction) in the spec. Therefore, I would propose we do an insertOne or some other basic write within each transaction (between start and abort/commit).

Additionally, we may want another test to demonstrate an error when executing on a 4.0+ standalone (there are appropriate "skip_if" functions for that, too).

Reply:
Replying to this and the above comment. This test does not require a working replicaset as it doesn't actually run a transaction. This test tests whether I could call the methods at all. libmongoc doesn't check whether there is a replica set (that's a server error), but it *does* test whether the server has the right wire protocol version (hence the ``< 4.0`` check).

Reply:
Noted. I'd suggest a comment to explain that. The title only said "mini test" so it wasn't clear what the intention was.

Even something like "ensure that methods can be called" as the title would add some more clarity.